### DISCUSSION TITLE
Choose either native pipe or magrittr pipe  with Cmd/Ctrl+Shift+M

### DISCUSSION THREAD

**Discussion by mark-andrews** [posit-dev/positron#5397] [Q&A] [answered]
I'm brand new to Positron so my apologies if this is simple or obvious. I searched here and elsewhere but could not find any mention of this topic.
I am using the RStudio keymaps in Positron so I can do Cmd/Ctrl+Shift+M in R or Quarto etc files to insert a pipe operator. It inserts a native pipe (`|>`). But is it possible to insert the magrittr pipe (`%>%`) instead?
In Rstudio, in Global Options / Code, you have a tick box that allows you to choose which pipe is used with Cmd/Ctrl+Shift+M. Is there anything like that in Positron?

**Comment by kv9898** [comment, marked as answer]
It's here in the settings! To get here, you can click on the gear icon in the bottom left, then choose "Settings", then in the settings search bar and type "Pipe". Hope this helps!

![image](https://github.com/user-attachments/assets/b79918a3-d350-4759-a49c-0359f58fe148)

**Reply by mark-andrews** [reply]
Thanks a lot @kv9898 . I obviously didn't search hard enough!
